functions/ingestion/ingestCSV.js

```diff
@@ -0,0 +1,122 @@
+const functions = require('firebase-functions');
+const Client = require('ssh2-sftp-client');
+const AdmZip = require('adm-zip');
+const sort = require('fast-sort');
+
+
+const ingestCSV = functions.https.onRequest((request, response) => {
```

Comment:
@iritush Would you pull this branch and confirm that you are able to run this function?  

Right now I'm not able to run this function, either on master or on this branch.  I get an error `Cannot read property 'host' of undefined` so I suspect this is an issue with accessing the config data.  If you are able to run this function locally with `npm run serve` then we can be confident that this PR is not hurting your code.

Thanks!

Reply:
I was able to run the 'ingestCSV' function in my environment but it appears that it is missing the code I added in my recent commits that had the parseCSVFromServer function. It was the changes I made in the parse-csv-issue74 branch, pull #129, and was merged by John Need into the master branch.